app/test_main.py

```diff
@@ -1,3 +1,21 @@
-from app.main import get_human_age
+from app.main import get_human_age, convert_to_human
 
-# write your code here
+
+def test_convert_to_human_age() -> None:
+    assert convert_to_human(14, 15, 9, 4) == 0
+    assert convert_to_human(15, 15, 9, 4) == 1
+    assert convert_to_human(23, 15, 9, 4) == 1
+    assert convert_to_human(24, 15, 9, 4) == 2
+    assert convert_to_human(27, 15, 9, 4) == 2
+    assert convert_to_human(28, 15, 9, 4) == 3
+    assert convert_to_human(100, 15, 9, 4) == 21
+
+
+def test_get_human_age() -> None:
+    assert get_human_age(14, 14) == [0, 0]
+    assert get_human_age(15, 15) == [1, 1]
+    assert get_human_age(23, 23) == [1, 1]
+    assert get_human_age(24, 24) == [2, 2]
+    assert get_human_age(27, 27) == [2, 2]
+    assert get_human_age(28, 28) == [3, 2]
+    assert get_human_age(100, 100) == [21, 17]
```

Comment:
Consider using `@pytest.mark.parametrize` to reduce repetition in your test cases. This will help improve maintainability and readability of your tests as required by the checklist.